packages/cli/src/commands/account/list.ts

```diff
@@ -8,24 +8,32 @@ export default class AccountList extends BaseCommand {
 
   static flags = {
     ...BaseCommand.flags,
+    local: flags.boolean({
+      default: false,
+      description: 'If set, only show local and hardware wallet accounts',
+    }),
   }
 
   requireSynced = false
 
   async run() {
-    this.parse(AccountList)
+    const res = this.parse(AccountList)
 
     const celoProvider: CeloProvider = this.kit.web3.currentProvider as any
-    const addresses = await this.kit.web3.eth.getAccounts()
+
+    // Retreive accounts from the connected Celo node.
+    const addresses = []
+    if (res.flags.local) {
+      addresses.push(...(await this.kit.web3.eth.getAccounts()))
+    }
```

Comment:
This could be easier to read in the future (and probably you want the !local for retrieving the node)
```suggestion
    const addresses = res.flags.local? [] : await this.kit.web3.eth.getAccounts()
```

Reply:
Does https://github.com/celo-org/celo-monorepo/pull/3830/commits/c24e42aa5cf206d410b7c301b8513935ed585699 address this comment?